blocks/api/utils.js

```diff
@@ -40,3 +42,45 @@ export function isUnmodifiedDefaultBlock( block ) {
 		isEqual( newDefaultBlock.attributes[ key ], block.attributes[ key ] )
 	);
 }
+
+/**
+ * Array of icon colors containing a color to be used if the icon color
+ * was not explicitly set but the icon background color was.
+ *
+ * @type {Object}
+ */
+const ICON_COLORS = [ '#191e23', '#fefdfe', '#7e70af' ];
```

Comment:
Minor: By convention, I've come to expect all module constants to be defined at the top of the file, immediately following dependencies. Should we consider that here?

Reply:
Where do these colors come from? Are we duplicating from stylesheet variables?

Reply:
The colors are from the stylesheet variables $dark-gray-900 and $light-gray-100. Before we used a scss importer but with our divisions in packages I'm not certain if this is something we should pursue now.